pkgs/tools/security/sequoia/octopus-librnp.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, rustPlatform, fetchFromGitLab, llvmPackages, pkg-config, openssl
+, sqlite, nettle, cargo, rustc, capnproto }:
```

Comment:
It's better to place args on a separate line each.

Reply:
True, fixed, thanks for the review :)